CryptoLightNews/MainTabBarController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  MainTabBarController.swift
+//  CryptoLightNews
+//
+//  Created by Richard Price on 15/12/2022.
+//
+
+import UIKit
+
+class MainTabBarController: UITabBarController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+        updateTabInsects()
+        updateTabColorScheme()
+        view.backgroundColor = .darkGray
+    }
+    private func setupNavControllers(with rootViewController: UIViewController, title: String, image: UIImage) -> UINavigationController {
+        let navController = UINavigationController(rootViewController: rootViewController)
+        rootViewController.title = title
+        navController.tabBarItem.title = title
+        navController.tabBarItem.image = image
+        return navController
+    }
+    private func setupUI() {
+        viewControllers = [
+            setupNavControllers(with: UIViewController(), title: "News", image: #imageLiteral(resourceName: "cryptoNews")),
+            setupNavControllers(with: UIViewController(), title: "Prices", image: #imageLiteral(resourceName: "prices_new"))
+        ]
+    }
+    private func updateTabInsects() {
+        guard let items = tabBar.items else {return}
+        for item in items {
+            item.title = ""
+            item.imageInsets = Sizes.tabInsets
+        }
+    }
+    private func updateTabColorScheme() {
+        tabBar.isTranslucent = true
+        tabBar.barTintColor = .clear
+        tabBar.tintColor = .systemGreen
+        UINavigationBar.appearance().prefersLargeTitles = true
+    }
+}
+enum Sizes {
```

Comment:
Sorry, maybe I didn't explain correctly. My fault. 
This enumeration should be inside the class. This way, you can have many of them with the same name, and they will not clash. So when we create the `NewsViewController`, for instance, we can have `Sizes`, too, with different constants, and the name won't clash with this one.
```suggestion
class MainTabBarController: UITabBarController {
   enum Sizes {
   ...
   }
   ...
}
```